app/src/main/java/id_iot/demo/ChosenRecipeFragment.java

```diff
@@ -0,0 +1,34 @@
+package id_iot.demo;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class ChosenRecipeFragment extends Fragment {
+
+    private Recipe chosenRecipe;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater,
+                             ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.chosen_recipe_fragment, container, false);
+
+        chosenRecipe = ((MainActivity) getActivity()).getChosenRecipe();
+        getActivity().setTitle(chosenRecipe.getRecipeName());
+
+        ImageView chosenRecipeImage = (ImageView) view.findViewById(R.id.chosen_recipe_image);
```

Comment:
I would recommend to keep declaration and usage as close as possible. Like declare chosenReciperImage and below set it Background image. It is a good practise to keep usage close to avoid searching for something. It is just faster to read